edb/lib/std/25-numoperators.edgeql

```diff
@@ -1381,11 +1472,11 @@ std::`//` (n: std::float64, d: std::float64) -> std::float64
 
 
 CREATE INFIX OPERATOR
-std::`//` (n: std::decimal, d: std::decimal) -> std::decimal
+std::`//` (n: std::decimal, d: std::decimal) -> std::bigint
```

Comment:
For similar reasons as for `ceil`, `floor` and `round` this should return `decimal`.